### ISSUE TITLE
Transform integer to string?

### ISSUE THREAD

**Issue by jhaneyrf** [jpmml/sklearn2pmml#467]
I built an XGBoost model with categorical variables that had string labels. I saved the levels for each variable.

Now my boss would like me to deploy this model as a PMML file on a third-party platform. At first, I could treat the categorical variables as strings and use CastTransformer to convert the strings to pandas CategoricalDtype(categories=str_levels[col]), and everything worked fine.

But it turns out that the third-party platform where we want to deploy the model doesn't allow string variables.

So I made a PMML based solely on the XGBoost model object. In comparing the two PMML files, I saw that not all of the levels in the categorical variables are used in the trees. That makes me wonder how I can accurately pass data into this file? If say the 4th out of 5 levels of a categorical variable are used in trees, will your code declare them in the data dictionary as having values 0, 1, 2, 4 or will it label them as 0, 1, 2, 3?

I also have tried developing a pipeline that would take integer inputs for these variables, use a LookupTransformer to convert them to strings and convert those strings to CategoricalDtype, but I can't seem to create a pipeline that produces correct output.

Can LookupTransformer handle integer inputs and string outputs? Or is there a better way to solve our problem?
